lib/PuppeteerSharp/IWebSocketOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Net.WebSockets;
+
+namespace PuppeteerSharp
+{
+    /// <summary>
+    /// Options for <see cref="WebSocket"/> creation.
+    /// </summary>
+    public interface IWebSocketOptions
```

Comment:
Do we have any class which is IWebSocketOptions but it's not IConnectionOptions?

Reply:
No, we don't. `IConnectionOptions` inherits `IWebSocketOptions`. `IWebSocketOptions` defines options for `WebSocketFactory` delegate, which is a subset of what is required by the `Connection` class. My preference was to keep `WebSocketFactory` delegate as independent as possible of the rest of the code.